packages/vite/src/node/server/ws.ts

```diff
@@ -1,5 +1,6 @@
 import chalk from 'chalk'
-import { Server } from 'http'
+import { Server, STATUS_CODES } from 'http'
+import { createServer, ServerOptions as HttpsServerOptions } from 'https'
```

Comment:
```suggestion
import { createServer as createHttpsServer, ServerOptions as HttpsServerOptions } from 'https'
```

Could be more clear this way when working without the context